packager/packager.cc

```diff
@@ -1027,24 +1019,26 @@ struct PackageInfoFinder {
             }
 
             auto compatibilityAnnotation = compatibilityAnnotationLit->asString();
-            if (compatibilityAnnotation != core::Names::strict() && compatibilityAnnotation != core::Names::legacy()) {
+            if (compatibilityAnnotation != core::Names::legacy()) {
                 if (auto e = ctx.beginError(send.loc, core::errors::Packager::InvalidConfiguration)) {
-                    e.setHeader("Argument to `{}` must be either 'strict' or 'legacy'", send.fun.show(ctx));
+                    if (compatibilityAnnotation == core::Names::strict()) {
+                        e.setHeader("strict has been deprecated as an argument to `{}`. If you wish to mark your "
+                                    "package as strictly path-based-autoloading compatible, do not provide an "
+                                    "autoloader_compatibility annotation",
```

Comment:
Don't put multiple sentences in a header

If you want to add more context, provide all the subsequent context with an `addErrorNote`.

Reply:
Ah sorry, sure.